File: py/867.py
class Solution:
    def transpose(self, matrix: List[List[int]]) -> List[List[int]]:
        rowCount = len(matrix)
        colCount = len(matrix[0])

        t = [[] for i in range(colCount)]

        print(rowCount)
        print(colCount)

        for i in range(colCount):
            for k in range(rowCount):
                 t[i].append(matrix[k][i])
        return t

File: cpp/867.cpp
class Solution {
  public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
      int rowCount = matrix.size();
      int colCount = matrix[0].size();
      std::vector<std::vector<int>> transposed(colCount);

      for (int i = 0; i < colCount; i++)
        for (int k = 0; k < rowCount; k++)
          transposed[i].push_back(matrix[k][i]);

      return transposed;
    }
};
